cmd/argo/commands/root.go

```diff
@@ -121,6 +121,7 @@ If your server is behind an ingress with a path (running "argo server --base-hre
 
 	client.AddKubectlFlagsToCmd(command)
 	client.AddAPIClientFlagsToCmd(command)
+	client.AddKubectlFlagsToArgoServerOpts()
```

Comment:
```suggestion
	client.AddKubectlFlagsToCmd(command)
	client.AddKubectlFlagsToArgoServerOpts()
	client.AddAPIClientFlagsToCmd(command)
```

same as above re: ordering